wisp/wisp.php

```diff
@@ -284,8 +284,9 @@ function wisp_CreateAccount(array $params) {
             if($userResult['meta']['pagination']['total'] === 0) {
                 $userResult = wisp_API($params, 'users', [
                     'email' => $params['clientsdetails']['email'],
-                    'first_name' => $params['clientsdetails']['firstname'],
-                    'last_name' => $params['clientsdetails']['lastname'],
+                    //set first_name and last_name if empty
+                    'first_name' => empty($params['clientsdetails']['firstname']) ? 'Unkown' : $params['clientsdetails']['firstname'],
+                    'last_name' =>empty($params['clientsdetails']['lastname']) ? 'User' : $params['clientsdetails']['lastname'],
```

Comment:
Would instead recommend to leave some space for syntactical reasons.
```
'last_name' =>empty($params['clientsdetails']['lastname']) ? 'User' : $params['clientsdetails']['lastname'],
```
Could be converted into:
```
'last_name' => empty($params['clientsdetails']['lastname']) ? 'User' : $params['clientsdetails']['lastname'],
```